python/sglang/srt/sampling_params.py

```diff
@@ -123,3 +123,17 @@ def normalize(self, tokenizer):
                 else:
                     stop_str_max_len = max(stop_str_max_len, len(stop_str))
             self.stop_str_max_len = stop_str_max_len
+
+    def to_srt_kwargs(self):
```

Comment:
Where it is used?

Reply:
before this change, i prepared my sampling params and call json.dumps(params.__dict__), and it will keep the member variable name 'stop_strs' instead of 'stop'. The Runtime.async_generate() pass it to backend, will cause errors.
after this change, i can call json.dumps(params.to_srt_kwargs) which will convert to correct dict.